beta/src/components/Breadcrumbs.tsx

```diff
@@ -17,9 +17,9 @@ function Breadcrumbs() {
             <div className="flex mb-3 mt-0.5 items-center" key={i}>
               <React.Fragment key={crumb.path}>
                 <Link href={crumb.path}>
-                  <a className="text-link dark:text-link-dark text-sm tracking-wide font-bold uppercase mr-1 hover:underline">
+                  <button className="text-link dark:text-link-dark text-sm tracking-wide font-bold uppercase mr-1 hover:underline">
```

Comment:
This doesn't look right. This is a link, not a button, right? `<NextLink>` is the thing that's supposed to add a route, unless I'm missing something?

Reply:
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md
So while I ran ESLint with old config it threw this error. We had no valid href for the anchor but used anchor tags. That's why I replaced it with button 🤔. Should I change everything to anchor tags and silence that rule like it was before ?

Reply:
BTW I changed those buttons to anchor tags and i guess that rule is not active by default in next ESlint config.